recipes/r-raceid3/meta.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: r-raceid3
+  version: 0.1.1
+
+source:
+  url: https://github.com/dgrun/RaceID3_StemID2_package/archive/master.zip
+  sha256: d73a9d5b031a34fee639103f683324d78c43b7f18eb834d289533c40afbbb57d
+
+build:
+  number: 0
```

Comment:
you probably need to also skip OSX as we skiped it for fateid as well

Reply:
oh I just saw you your commits -- ok will emulate